test/integration/controllers/quiz.spec.js

```diff
@@ -54,6 +54,7 @@ describe('Quiz controller', () => {
     currentUser = TokenService.validateAccessToken(accessToken)
 
     testQuiz = await app.post(endpointUrl).send(testQuizData).set('Authorization', `Bearer ${accessToken}`)
+    testQuizBody = testQuiz.body

```

Comment:
i think that herre you just can rename this variable and don't put all parameters of the body, only `id` if you use only `id`

Reply:
> i think that herre you just can rename this variable and don't put all parameters of the body, only `id` if you use only `id`

fixed